chains/solana/README.md

```diff
@@ -21,7 +21,7 @@ go install github.com/gagliardetto/anchor-go@v0.2.3
 anchor build
 
 # go bindings need to be regenerated if contract changes were made
-./scrips/anchor-go-gen.sh
+./scripts/anchor-go-gen.sh
```

Comment:
Has this basically never worked before? :scream:

Reply:
haha, it's just a readme!